src/equations/compressible_navier_stokes_3d.jl

```diff
@@ -163,11 +163,11 @@ function flux(u, gradients, orientation::Integer,
 
     # Diagonal parts
     # (4/3 * (v1)_x - 2/3 * ((v2)_y + (v3)_z)
-    tau_11 = 4.0 / 3.0 * dv1dx - 2.0 / 3.0 * (dv2dy + dv3dz)
+    tau_11 = 4.0f0 / 3 * dv1dx - 2.0f0 / 3 * (dv2dy + dv3dz)
     # (4/3 * (v2)_y - 2/3 * ((v1)_x + (v3)_z)
-    tau_22 = 4.0 / 3.0 * dv2dy - 2.0 / 3.0 * (dv1dx + dv3dz)
+    tau_22 = 4.0f0 / 3 * dv2dy - 2.0f0 / 3 * (dv1dx + dv3dz)
     # (4/3 * (v3)_z - 2/3 * ((v1)_x + (v2)_y)
-    tau_33 = 4.0 / 3.0 * dv3dz - 2.0 / 3.0 * (dv1dx + dv2dy)
+    tau_33 = 4.0f0 / 3 * dv3dz - 2.0f0 / 3 * (dv1dx + dv2dy)
```

Comment:
Maybe `4 * dv3dz /  3- 2 * (dv1dx + dv2dy) / 3` is better?

I have no idea which way goes better @ranocha

Reply:
I would prefer your suggestion from the comment since we will lose precision otherwise:
```julia
julia> 4.0f0 / 3 * 1.0
1.3333333730697632

julia> 4.0f0 * 1.0 / 3
1.3333333333333333
```

Reply:
Make sense.